_includes/home-page.html

```diff
@@ -162,56 +159,73 @@ <h1 class="used-by">Used By</h1>
 					<div class="company-image">
 						<img src="./assets/images/company-logo/hpe-logo-color-no-trim.svg" alt="Hewlett Packard"
 							loading="lazy" id="logo-dark-light"
-							data-logo-for-dark="./assets/images/company-logo/hpe-logo-dark.svg"
+							data-logo-for-dark="./assets/images/company-logo/hpe-logo-dark.png"
```

Comment:
Why `.png` here when we have the same `.svg`?

@captain-Akshay

Reply:
Oh this was because of the css was not being applied on some svg that's why someplace we used the png but this issue is resolved by Lee so i will be closing it